runtime/oti/j9nonbuilder.h

```diff
@@ -4047,6 +4047,7 @@ typedef struct J9JITConfig {
 	void ( *jitMethodBreakpointed)(struct J9VMThread *currentThread, struct J9Method *method) ;
 	void ( *jitMethodUnbreakpointed)(struct J9VMThread *currentThread, struct J9Method *method) ;
 	void ( *jitIllegalFinalFieldModification)(struct J9VMThread *currentThread, struct J9Class *fieldClass);
+	void* compilationRuntime;
```

Comment:
Could you please check the spacing. It's hard to judge from the web-browser.
VM coding conventions require to insert a TAB (not spaces). I believe that lines 4047-4049 suffer from the same issue.